scripts/02_retrieve_metadata.py

```diff
@@ -178,6 +178,11 @@ def write_output(outputs, configurations, output_dir, configuration_space, metri
             fh.write("%s: %s\n" % (key, description[key]))
 
 
+class DummyDatamanager():
+    def __init__(self, info):
+        self.info = info
+        self.feat_type = {"A1": "numerical"}
```

Comment:
This appears to be a bad idea. This should be created per task to create correct configuration spaces when reading in configurations from disk/meta-learning.